.github/workflows/tests_windows.yml

```diff
@@ -109,8 +116,53 @@ jobs:
     outputs:
       exec_model: ${{ steps.exec_model.outputs.exec_model }}
       kokkos_version: ${{ steps.kokkos_version.outputs.kokkos_version }}
+  
+
+  build_lapack:
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [windows-latest]
+    timeout-minutes: 30
+    name: LAPACK-Reference
+    runs-on: ${{ matrix.os }}
+
+    steps:
+      - name: Cache installation directories
+        id: lapack-cache
+        uses: actions/cache@v3
+        with:
+          path: D:\a\install_dir\lapack
+          key: ${{ matrix.os }}-lapack-Debug
+      
+      - name: Create installation directory
+        if: steps.lapack-cache.outputs.cache-hit != 'true'
+        run: |
+          Remove-Item -Path D:\a\install_dir\lapack -Recurse -Force -ErrorAction Ignore
+          mkdir -p D:\a\install_dir\lapack
+
+      - name: Setup msys2
+        if: steps.lapack-cache.outputs.cache-hit != 'true'
+        uses: msys2/setup-msys2@v2
+        with:
+          msystem: MINGW64
+          update: true
+          install: >-
+            mingw64/mingw-w64-x86_64-gcc-fortran
+
+      - name: Install lapack
+        if: steps.lapack-cache.outputs.cache-hit != 'true'
+        run: |
+          cd D:\a
+          git clone --quiet --recursive https://github.com/Reference-LAPACK/lapack.git
+          cd lapack
+          mkdir build
+          cd build
+          cmake -G"MSYS Makefiles" -DBUILD_SHARED_LIBS=ON -DCMAKE_INSTALL_PREFIX=D:\a\install_dir\lapack ..
+          cmake --build . -j --target install
```

Comment:
We can install it with vcpkg. I tried to install it with vcpkg but turn to install with the instructions above since the path to lib file is not clear.